### ISSUE TITLE
[req] installer should terminate active Jackett

### ISSUE THREAD

**Issue by cloudrootab** [Jackett/Jackett#12780]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Using HTTP Client: HttpWebClient2
2022-01-07 09:09:10	Info	
Using FlareSolverr: No
2022-01-07 09:09:10	Info	
Using proxy: Disabled
2022-01-07 09:09:10	Info	
App config/log directory: C:\ProgramData\Jackett
2022-01-07 09:09:10	Info	
ThreadPool MaxThreads: 1023 workerThreads, 1000 completionPortThreads
2022-01-07 09:09:10	Info	
Running in Docker: No
2022-01-07 09:09:10	Info	
Jackett variant: CoreWindows
2022-01-07 09:09:10	Info	
OS version: Microsoft Windows NT 10.0.22000.0 (64bit OS)
2022-01-07 09:09:10	Info	
Environment version: 6.0.1 (C:\ProgramData\Jackett\)
```

### Description

When running an upgrade via winget. It always fails on Jackett with an error "Access is denied", in regards to Accessibility.dll. 
I then have to kill the tray icon and the service manually. And then do the update again.
My guess here, is that the installation program doesn't shutdown the processess like it should. 

Kudos, and thanks!

### Logged Error Messages

See the attached screenshot

### Screenshots

![image](https://user-images.githubusercontent.com/62222189/148578549-6fd2e183-e87a-4d9b-b881-13d74082186c.png)

**Comment by alkuzad** [comment]
https://jrsoftware.org/ishelp/index.php?topic=setup_closeapplications ? Don't have visual studio to build and test this out

**Comment by ngosang** [comment]
@ilike2burnthing The installed configuration is defined in this file => https://github.com/Jackett/Jackett/blob/master/Installer.iss#L59
We are already closing some process on uninstall. Maybe we can copy those lines in the "Run" section, line 51 ?

**Comment by garfield69** [comment]
v0.20.363

**Comment by cloudrootab** [comment]
Similar error still present.
v 0.20.371.0 from winget
![image](https://user-images.githubusercontent.com/62222189/149930887-c6ee8779-70c5-476a-81d3-fc03cb665207.png)

**Comment by ilike2burnthing** [comment]
If you manually run the [installer](https://github.com/Jackett/Jackett/releases/download/v0.20.371/Jackett.Installer.Windows.exe), does it work? Does it work if you run it with the `/silent` parameter?

If you check in Task Manager before and during the winget run, are any of the Jackett processes terminated?

**Comment by cloudrootab** [comment]
> If you manually run the [installer](https://github.com/Jackett/Jackett/releases/download/v0.20.371/Jackett.Installer.Windows.exe), does it work? Does it work if you run it with the `/silent` parameter?
> 
> If you check in Task Manager before and during the winget run, are any of the Jackett processes terminated?

I'll check upon next update-cycle

**Comment by ilike2burnthing** [comment]
In case you don't want to wait, you could install an older version, e.g. [v0.20.363](https://github.com/Jackett/Jackett/releases/download/v0.20.363/Jackett.Installer.Windows.exe), and then update.

**Comment by cloudrootab** [comment]
Did a manual downgrade (363), followed by upgrades as requested. 
Manually running the installer doesn't kill processes as expected. Even if it prompts if it should kill the tasks. _But_ when choosing to cancel the installation, JackettService and JackettConsole are then closed. JackettTray is still active.

Using the /silent-flag, results in the same behaviour. Just without the prompt.

**Comment by ilike2burnthing** [comment]
So you get this screen:
![setup](https://user-images.githubusercontent.com/59480337/149685284-2160e01e-17a7-48c9-bd30-a74def7d3634.png)
You leave it on `Automatically close the applications`, click `Next`, it says `Closing applications...`, but then fails to do so?

During that time, `JackettConsole`, `JackettService`, and `JackettTray` are all still running in Task Manager, and only when you click `Cancel installation` are the first 2 terminated (along with the installer), but `JackettTray` remains active?

**Comment by cloudrootab** [comment]
> So you get this screen: ![setup](https://user-images.githubusercontent.com/59480337/149685284-2160e01e-17a7-48c9-bd30-a74def7d3634.png) You leave it on `Automatically close the applications`, click `Next`, it says `Closing applications...`, but then fails to do so?
> 
> During that time, `JackettConsole`, `JackettService`, and `JackettTray` are all still running in Task Manager, and only when you click `Cancel installation` are the first 2 terminated (along with the installer), but `JackettTray` remains active?

Correct

**Comment by ilike2burnthing** [comment]
Download this [test build](https://dev.azure.com/Jackett/0f0cb3d0-9b6b-4a86-876d-0c2e17c18801/_apis/build/builds/5179/artifacts?artifactName=drop&api-version=7.0&%24format=zip), extract and run the installer, and see if that resolves your issue.

**Comment by cloudrootab** [comment]
TL;DR: Looks good

Tried multiple times, with downgrades to 371. Followed by manual installs of the v380. Although it didn't prompt about killing running processes. It went on with the installation without any (prompted) problems. And the same with the /silent flag as well. 
I'd say that it's working as expected. And hopefully this will translate over to the winget upgrade process as well.

**Comment by garfield69** [comment]
v0.20.399
